cdap-app-templates/cdap-etl/cdap-etl-core/src/main/java/co/cask/cdap/template/etl/common/TrackedTransform.java

```diff
@@ -41,4 +42,11 @@ public void transform(IN input, Emitter<OUT> emitter) throws Exception {
     metrics.count("records.in", 1);
     transform.transform(input, emitter);
   }
+
+  @Override
+  public void destroy() {
+    if (transform instanceof Destroyable) {
+      Destroyables.destroyQuietly((Destroyable) transform);
```

Comment:
Yes, will delegate here. Was there a reason the ``TransformExecutor`` destroys quietly though?